.gitattributes

```diff
@@ -0,0 +1 @@
+dist/** -diff linguist-generated=true
```

Comment:
This is to make sure it hides the dist assets in the diff: as you can see in the "Files changed" tab, the diff of the 2 dist files are hidden by default (`Some generated files are not rendered by default.`).